docs/src/content/docs/reference/frontmatter.md

```diff
@@ -119,6 +119,9 @@ hero:
     - text: View on GitHub
       link: https://github.com/astronaut/my-project
       icon: external
+      attrs:
+        target: _blank
+        rel: noopener
```

Comment:
I’m a little bit torn about showing this as in general opening in a new window for this link in this example is not really considered a best practice (see https://www.nngroup.com/articles/new-browser-windows-and-tabs/ for example for some UX research), although I know plenty of people will still want to do exactly this.

Something like this could make sense:

```suggestion
    - text: View on GitHub
      link: https://github.com/astronaut/my-project
      icon: external
      attrs:
        rel: me
```

The other link attribute that could make sense sometimes would be `download`, but we’re missing a good icon for that one.